src/transformers/models/hubert/modeling_hubert.py

```diff
@@ -603,6 +604,39 @@ def forward(self, hidden_states, attention_mask=None, output_attentions=False):
         return outputs
 
 
+# Copied from transformers.models.wav2vec2.modeling_wav2vec2.Wav2Vec2AttnAdapterLayer with Wav2Vec2->Hubert
+class HubertAttnAdapterLayer(nn.Module):
```

Comment:
Note that the changes here and in all other modeling files that are != modeling_wav2vec2.py are just added so that the # Copied from logic can stay in place